specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2019-08-08/storageaccounts.json

```diff
@@ -411,13 +524,28 @@
       "type": "string",
       "x-ms-parameter-location": "method"
     },
+    "NewAccountNameParameter": {
+      "description": "New storage account name when doing undelete storage account operation.",
+      "name": "newAccountName",
+      "in": "query",
+      "required": false,
```

Comment:
If this is false does that mean it will reuse the existing account name?

Reply:
yes, if client side does not specify the new account name, it will re-use the old account name.